### PR TITLE
Players Gallery Page

### PR DESCRIPTION
**Pull request by perez-e**
Screenshot 2023-07-21 at 10 33 09 AM

### CODE REVIEW

File: packages/web/lib/player.ts [hunk 1]
```diff
@@ -69,3 +70,23 @@ export const getPlayerSplits = async (props: {
   const data = (await response.json()) as GameraPlayerSplits
   return data
 }
+
+export const getPlayerGalleryList = async (league?: string) => {
```
Comment (perez-e) [inline packages/web/lib/player.ts:74]: I wasn't sure if I should include this function here or in @statmuse/core. Any thoughts?

File: packages/web/src/components/player-gallery.astro
Comment (perez-e) [inline packages/web/src/components/player-gallery.astro:1]: Maybe i should move this component over to @layouts

Comment (adamelmore) [reply packages/web/src/components/player-gallery.astro:1]: it's just used the one time, right?

File: packages/web/lib/player.ts (hunk 1 above)
Comment (adamelmore) [reply packages/web/lib/player.ts:74]: my heuristic has been, "will this code be used by something other than the web frontend?". if not, i put it in the web lib dir; if so, i put it in core. i've probably broken my rule some, though.

File: packages/web/src/components/player-gallery.astro
Comment (perez-e) [reply packages/web/src/components/player-gallery.astro:1]: it's used in `/players` and `/[league]/players` pages

File: packages/web/lib/player.ts (hunk 1 above)
Comment (perez-e) [reply packages/web/lib/player.ts:74]: cool. I'll leave in web/lib then

File: packages/web/src/pages/players.astro [hunk 2]
```diff
@@ -1 +1,10 @@
-players
+---
+import PlayerGallery from '@components/player-gallery.astro'
+import { getPlayerGalleryList } from '@lib/player'
+
+const { league } = Astro.params
```
Comment (adamelmore) [inline packages/web/src/pages/players.astro:5]: there isn't a league in this case, right?

**Review by adamelmore — APPROVED** [review]

**Pull request merged**